README.md

```diff
@@ -165,7 +177,30 @@ managers, and EXPLORE opportunities for aggregation, management, visualization,
 
 <br />
 
-# <b>Projects</b>
+## Technology Transfer
+
+>### **National Science Foundation Innovation Corps (NSF I-Corps) Training Program**
+> The U.S. National Science Foundation’s Innovation Corps (I-Corps™) program is an immersive, entrepreneurial training program that facilitates the transformation of invention to impact. This immersive, seven-week experiential training program prepares scientists and engineers to extend their focus beyond the university laboratory — accelerating the economic and societal benefits of NSF-funded and other basic research projects that are ready to move toward commercialization. Visit the [NSF's I-Corps website](https://new.nsf.gov/funding/initiatives/i-corps) for more information.
+>#### **Desert and Pacific Regional Hub 2023 - Cohort 6 - Fire Data Dynamics**
+>AIR's technology team is working with Tech Launch Arizona on licensing for our Fire Data Dynamics prototype to help make meaningful impacts in fire prediction, response, and analysis. As part of this licensing process, we participated in NSF's I-Corps where we learned about the Lean Startup Methodology which has a strong focus on customer discovery and defining customer segments.
+
+<br />
+
+## Projects
+
+>### **Fire Data Dynamics**
```

Comment:
For this and the following project, I realize we didn't add any pictures to the markdown. I could go either way on this, as I don't want this page to get too bloated, but figured I'd bring it up just in case.

Reply:
Good catch. I just added the image to markdown to keep uniformity.